client_test.go

```diff
@@ -1916,7 +1918,15 @@ func TestLatency(t *testing.T) {
 
 		returnedLatency = returnedLatency.Round(time.Millisecond)
 
-		if returnedLatency != expectedLatency {
+		latencyDiff := func() time.Duration {
+			diff := returnedLatency - expectedLatency
+			if diff < 0 {
+				return -diff
+			}
+			return diff
+		}()
+
+		if latencyDiff > time.Millisecond*3 {
```

Comment:
This is so flakey... I don't like this at all.

Reply:
Yeah true but I don't see an alternative, I was getting 0ms locally and it worked consistently, but when actions runs the test it fails. Do you have a better idea?

Reply:
I could try reverting the last 2 commits and using `returnedLatency.Truncate` instead of`returnedLatency.Round`, but even if that works now I'm not sure it won't eventually cause your tests to fail because actions ran too slowly.